tests/MultipleAuthSchemes/Controllers/HomeB2CController.cs

```diff
@@ -24,7 +24,7 @@ public HomeB2CController(ILogger<HomeController> logger, IDownstreamWebApi downs
             ScopeKeySection = "DownstreamB2CApi:Scopes", UserFlow = "b2c_1_susi", AuthenticationScheme = "B2C")]
         public async Task<IActionResult> Index()
         {
-            var value = await _downstreamWebApi.GetForUserAsync<Task>("DownstreamB2CApi", "", null, null, "B2C");
+            var value = await _downstreamWebApi.GetForUserAsync<Task>("DownstreamB2CApi", string.Empty, authenticationScheme:"B2C");
```

Comment:
the string.Empty is probably not even necessary? is it?

Reply:
there is no default for the relative path.